src/app/components/reports/reports.component.html

```diff
@@ -0,0 +1,60 @@
+<form id="form" [formGroup]="schedulerForm" (ngSubmit)="onSubmit()">
+  <label class="formLabel">Select TFR</label>
+  <mat-form-field class="mat-selector" appearance="fill">
+    <mat-label>Select TFR</mat-label>
+    <mat-select formControlName="tfr">
+      <mat-option *ngFor="let tfr of tfrList" [value]="tfr.project_id">
+        {{ tfr.project_id }}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <label class="formLabel">Select Template</label>
+  <mat-form-field class="mat-selector" appearance="fill">
+    <mat-label>Select Template</mat-label>
+    <mat-select formControlName="type">
+      <mat-option *ngFor="let template of templates" [value]="template.value">
+        {{ template.viewValue }}</mat-option
+      >
+    </mat-select>
+  </mat-form-field>
+
+  <app-frequency-picker></app-frequency-picker>
+
+  <label class="formLabel">Who do you wish to send reports to?</label>
```

Comment:
Please create variable for this and then should  add interpolation here 
Why we need this : because if future if someone wanted to change label they not need to search sender text in html this will be really pathetic so search in HTML if you have 10K lines of code for example